samples/valueentity-counter/pom.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.example</groupId>
+  <artifactId>valueentity-counter</artifactId>
+  <version>0.1</version>
+  <packaging>jar</packaging>
+
+  <name>first-service</name>
+    <properties>
+
+    <!-- TODO Update to your own Docker repository or Docker Hub scope -->
+    <dockerImage>my-docker-repo/${project.artifactId}</dockerImage>
+    <dockerTag>${project.version}</dockerTag>
+    <mainClass>com.example.Main</mainClass>
+
+    <jdk.target>11</jdk.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <akkaserverless-sdk.version>0.7.0-beta.10</akkaserverless-sdk.version>
```

Comment:
SDK and proxy are not released together?
https://github.com/lightbend/akkaserverless-java-sdk/pull/11/files#diff-b8da8ef2298c559c0f88129f0dd2310d0afb2d97c9cbcb783d97659322f72e08R4

Reply:
No, they are now apart. Two different repos.